src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj

```diff
@@ -44,6 +44,11 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+		<Page Remove="Themes\Generic.xaml" />
```

Comment:
👀

Reply:
That's a required pattern to fix a VS/MSBuild bug ;-)

Reply:
![ImaginationSpongebobSquarepantsGIF](https://user-images.githubusercontent.com/1075116/107437800-d40a1e80-6b2f-11eb-8abb-0caef3dab7aa.gif)